Engine/DataFeeds/Transport/LocalFileSubscriptionStreamReader.cs

```diff
@@ -75,10 +96,6 @@ public void Dispose()
                 _streamReader.Dispose();
                 _streamReader = null;
             }
-            if (_zipFile != null)
-            {
-                _zipFile.Dispose();
```

Comment:
I think it's important to document this behavior, specifically that the `LocalFileSubscriptionStreamReader` opens disposable items (zip files) and doesn't dispose of them... which means he assumes someone else will dispose of these items. Who does?